package.json

```diff
@@ -73,6 +74,7 @@
     "style-loader": "^0.13.1",
     "svg4everybody": "^2.1.0",
     "svgo": "^0.6.6",
+    "throttle-debounce": "^1.0.1",
```

Comment:
we generally use `{ throttle, debounce }` from 'lodash` - if you can replace it easily, for consistency?